src/Microsoft.AspNet.Mvc.ModelBinding/Internal/ModelBindingHelper.cs

```diff
@@ -117,5 +119,47 @@ internal static bool IsFormatException(Exception ex)
             }
             return false;
         }
+
+        internal static object ConvertValuesToCollectionType<T>(Type modelType, IList<T> values)
```

Comment:
`public` - there's a very very high bar for marking any method or type as internal. If you needed to replace our `[FromHeader]` binder, you'd need this method to be available.